### ISSUE TITLE
Fix flake8 issues

### ISSUE THREAD

**Issue by lwasser** [earthlab/matplotcheck#280]
many new pr's are failing due to flake8 issues on master. Let's do the following

1. clean up all flake8 issues in a new pr and get that merged
2. make sure our pre commit hooks are working. i'm not sure how this slipped throug CI?
```
./matplotcheck/vector.py:393:38: E741 ambiguous variable name 'l'
./matplotcheck/vector.py:455:49: E741 ambiguous variable name 'l'
./matplotcheck/base.py:50:17: E741 ambiguous variable name 'l'
./matplotcheck/base.py:71:17: E741 ambiguous variable name 'l'
./matplotcheck/base.py:485:32: E741 ambiguous variable name 'l'
./matplotcheck/base.py:487:32: E741 ambiguous variable name 'l'
./matplotcheck/base.py:676:37: E741 ambiguous variable name 'l'
./matplotcheck/base.py:789:21: E741 ambiguous variable name 'l'
./matplotcheck/base.py:802:25: E741 ambiguous variable name 'l'
./matplotcheck/base.py:987:17: E741 ambiguous variable name 'l'
./matplotcheck/base.py:1102:13: E741 ambiguous variable name 'l'
./matplotcheck/timeseries.py:107:45: E741 ambiguous variable name 'l'
./matplotcheck/tests/test_lines.py:169:47: E741 ambiguous variable name 'l'
```

@nkorinek can you please work on this today? it should be a quick fix.

**Comment by nkorinek** [comment]
This can be closed, fixed in #274 @lwasser

**Comment by lwasser** [comment]
that pr really closed a lot of issues. thank you @nkorinek closing!
